packages/apm/src/span.ts

```diff
@@ -153,6 +153,19 @@ export class Span implements SpanInterface, SpanContext {
     return span;
   }
 
+  /**
+   * @inheritDoc
+   */
+  public getTransaction(callback: (transaction: Transaction) => void): void {
```

Comment:
Please please please, let's not have a `getFoo` that does not return `Foo`, this causes unnecessary confusion.